clients/netcore/TrustAPIClient.gen.cs

```diff
@@ -734,6 +757,9 @@ public System.Threading.Tasks.Task<SaveValueCreateProofResult> SaveValueAsync(bo
         /// <exception cref="ApiException">A server side error occurred.</exception>
         public async System.Threading.Tasks.Task<SaveValueCreateProofResult> SaveValueAsync(bool? skipUniqueConstraintViolation, Precision? precision, ProofKind? proofKind, CreateValueByteModel body, System.Threading.CancellationToken cancellationToken)
```

Comment:
The Trust-API has controller actions which call the Data-API underneath and thus share certain parameters like the `skipUniqueConstraintViolation`.
For a meaningful update of the API clients to 1.8.X we should first update the Trust-API and than re-generate the Trust-API client.

Since we are in a "chicken or the egg" kind of dilemma for doing this we can either create a nuget package locally for the Data-API und use it for updating the Trust-API or release the 1.8.0-rc0, update Trust-API (with the Data-API 1.8.0-rc0 client) and create yet another release 1.8.0-rc1 afterwards.

@tributech-dhofer What do you prefer?

Reply:
Was solved with option one.